packages/core/src/utils/getSCMigrationRef/index.js

```diff
@@ -0,0 +1,2 @@
+import getSCMigrationRef from './getSCMigrationRef'
+export default getSCMigrationRef
```

Comment:
How do you feel about `export { default } from './getSCMigrationRef'`?

Reply:
Oh is that available? Wasn't sure if we could use `export default from`